### ISSUE TITLE
Enhancement Request: Adblock Extension

### ISSUE THREAD

**Issue by flywire** [Ansuel/tch-nginx-gui#379]
Couple of potential openwrt candidates:

- https://github.com/stangri/openwrt_packages/blob/master/simple-adblock/files/README.md

- https://github.com/openwrt/packages/tree/master/net/adblock/files

**Comment by FrancYescO** [comment]
did you tried `luci-app-adblock`?

**Comment by flywire** [comment]
No, I've been messing with leds for far too long, I'll try the ones above. Any recommendations? Luci is nice but I'd prefer to use TG799 resources for something else. I was more thinking of the TG800.

btw Is luci working on TG800?

**Comment by xes** [comment]
Then.. luci adblock for chaos calmer is really old. 
Recent versions include many more lists. 
Maybe it be possible to port some 3.x.x version of that module.

**Comment by flywire** [comment]
https://whrl.pl/RfW5Gh

> I sussed out getting automated ad-blocking working on the DJA0231 ...

**Comment by Sevio007** [comment]
Thanks for linking my post Flywire :)
I had asked for the enhancement in the "issues" thread previously, but got no response, had no idea there was already a thread about this lol.

**Comment by FrancYescO** [comment]
Probably more than overloading the dnsmasq conf a solution like using  https://adguard.com/ will just avoid having the ram full

**Comment by Sevio007** [comment]
Actually using the lighter "spark" list frees up my ram to like 110mb now.

**Comment by Sevio007** [comment]
If you want to white list any domains just add it to the regular dnsmasq.conf file in the format:

server=/google.com/www.google-analytics.com/202.142.142.142 (The DNS you want it to go through instead, this one being my providers DNS).

Suggested white list:
https://discourse.pi-hole.net/t/commonly-whitelisted-domains/212

**Comment by Sevio007** [comment]
Steven blacks block list in the correct dnsmasq format...

wget -O- https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts | awk '$1 == "0.0.0.0" { print "address=/"$2"/0.0.0.0/"}' > /etc/dnsmasq.conf.ads

**Comment by Sevio007** [comment]
White list excel file:

[White list.xlsx](https://github.com/Ansuel/tch-nginx-gui/files/3765990/White.list.xlsx)

**Comment by FrancYescO** [comment]
for ARM devices: https://www.macoers.com/blog/antonio-macolino/adblock-su-dga4130-dga4131-dga4132-homeware-18-rimuovere-pubblicita
